js-core/week-04/lesson.md

```diff
@@ -18,26 +21,37 @@ This class is based on the
 
 ## Getting Started
 
-1. Download and install the latest Current version of NodeJS from
-   [here](https://nodejs.org/en/download/current/)
-
-To test that it was installed and running properly, go to your terminal and run
-the command: `node -v` You should get the node version printed on your terminal,
-for example, `v7.3.0`
-
-1. Fork the
+1. Download and install the latest Current version of NodeJS.
+  * Windows and Max users: Download the *installer* (.msi or .pkg) from
+  [here](https://nodejs.org/en/download/)
+  * Linux (Ubuntu) users: you can run the following script in your terminal:
+```Shell
+  curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+  sudo apt-get install -y nodejs
+```
+  This will require you to enter your password and will configure your computer
+  automatically. You can find the full instructions, if necessary,
+  [here](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions).
+  A note on security: don't execute a script on your terminal if you don't
```

Comment:
Might be considered a bit superfluous at this stage...I still think it's never too early to start thinking about security :)